src/checks/resource.projectTitle.controlled-1.0.0.xml

```diff
@@ -32,13 +32,12 @@
       projectTitle <- trimws(projectTitle)
       
       # Read in the controlled project title list
-      projectListFn <- "ess-dive-projects.json"
-      projectListPath <- sprintf("%s/%s", mdq_params[['metadigDataDir']], projectListFn)
+      projectListPath <- "https://mule.ess-dive.lbl.gov/api/project/"
```

Comment:
I assume this works because `read_json` will read from a URL rather than a file. When this was a local file, it is not a problem that it is read multiple times (once for every checked dataset, so probably more than a million times for the dataone corpus, but probably a lot fewer for ESS-DIVE). But now, as a network retrieval, the efficiency goes way down. Would it be reasonable to cache the file locally and only retrieve it once a day or something to cut down on the number of network calls? Also, that would help by making the project list consistent for a set of runs that are close together in time.  Did you ever decide how changes in the project list should trigger re-runs of the suite for all datasets?